_data/accessibility-tests/icon.yml

```diff
@@ -0,0 +1,61 @@
+title: Icon
+component_status: pass
+test_items:
+# General items
+  - summary: Color is not the only method used to convey icon information.
+    summary_additional: |
+      When you view an icon that represents an action and determine the action,
+      the text (not color alone) conveys the icon's meaning where necessary.
+    test_status: pass
+    test_type: general
+    version_tested: 3.7.1
+    wcag_criterion: 1.4.1
+  - summary: Icons meet color contrast requirements.
+    summary_additional: |
+      When you view the icon on a webpage
+      and use ANDI or color contrast analyzer to look at the hex colors,
+      contrast between the icon and background is at least 3:1.
+    test_status: pass
+    test_type: general
+    version_tested: 3.7.1
+    wcag_criterion: 1.4.11
+# Zoom/screen magnification items
+# Keyboard navigation items
+  - summary: Focus indicator is clearly visible on interactive icons.
+    summary_additional: |
+      When you use a keyboard to tab into an interactive icon and the icon receives focus,
+      you see a visible indicator around the icon.
```

Comment:
Off the top of my head, I'm thinking sometimes we might use one over the other for the purposes of plain language and brevity. As in the case above, what we have is shorter and hopefully simpler than the updated version. The question becomes: which is easier for the user to understand? Is the current version not clear enough? If it isn't, then I'd go with the update.

Reply:
I'll leave it as is for now. In the future, it would be great to make statements like these consistent across components whenever possible, but it doesn't need to be now.